java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -17,6 +17,7 @@ public class TransactionTestUtilities {
     private static final String key5 = "{key}" + UUID.randomUUID();
     private static final String key6 = "{key}" + UUID.randomUUID();
     private static final String key7 = "{key}" + UUID.randomUUID();
+    private static final String setKey2 = "{key}" + UUID.randomUUID();
```

Comment:
couple questions:
- is setKey1 being added in a separate PR?
- setKey2 is more specific than keyX but since all the other set keys follow the format keyX should we do the same? Its definitely a minor thing but it looks slightly weird to have a bunch of set keys called keyX and also have some set keys called setKeyX. Either way should be fine, just thought I'd point it out

Reply:
`key7` is `setKey1`, but I keep all renaming existing keys in another PR (#160)

Reply:
assuming this PR will be merged before #160, should we just follow the existing convention in this PR and then rename this key together with all the others in #160?

Reply:
How to name it? `key7.5`?

Reply:
I just meant that if we merge this PR before [#1284](https://github.com/aws/glide-for-redis/pull/1284) maybe we should follow the existing convention and name it key8 here, then update it to setKey2 in #1284. Not a big deal though